book/section3/wallets.md

```diff
@@ -1,12 +1,37 @@
+Wallets are an essential component on every Self-Sovereign Identity interaction, and as you might have guessed, just like in the physical world a wallet holds your identifiers (IDs), in the digital SSI realm their function is to store and manage Decentralized Identifiers (DIDs), Verifiable Credentials (VCs), cryptographic keys, and other related assets.
 
-We will talk about creating and recovering Identus wallets.
+Since many SSI frameworks rely on a Blockchain to publish DIDs, there is a common misconception that SSI wallets work in a similar way. Although both SSI and Blockchain wallets require a `seed phrase` built from a random set of `mnemonics`, thats where the overlap ends, because the balance and history of a Blockchain wallet can be restored from the ledger itself as opposed to an SSI wallet, where all the stored information exists only on the device and needs to be manually backed up and restored.
 
-Tips for using:
+In essence, a wallet in SSI is piece of software that allows users to store, manage, and present proof of their digital identities and credentials. It acts as a repository for digital assets required to fulfill every SSI interaction, ensuring that users and entities have complete control over their data. In the case of Identus, the [Edge Agent SDK](https://github.com/hyperledger/identus-edge-agent-sdk-ts) provides all the abstractions needed to operate a wallet by an individual on a browser or mobile app, and the [Cloud Agent](https://github.com/hyperledger/identus-cloud-agent), provides a `REST API` to operate a wallet in the cloud, either to itself (single tenant) or for third-parties (multi tenant) as a custodial wallet.
 
-- Pluto Encrypted
-- Keycloak for OAuth / OpenIDConnect
\ No newline at end of file
+## Edge Agent SDK in Identus
+
+Identus provides their wallet interface trough the Edge Agent SDK component available in 3 flavors:
+
+- [TypeScript](https://github.com/hyperledger/identus-edge-agent-sdk-ts) for Web and Node apps.
+- [Swift](https://github.com/hyperledger/identus-edge-agent-sdk-swift) for iOS and Mac.
+- [Kotlin](https://github.com/hyperledger/identus-edge-agent-sdk-kmp) for Android and JVM.
+
+Each of the flavors provide the same building blocks implementations:
+
+- Apollo: Provides a suite of necessary cryptographic operations.
+- Castor: Provides a suite of operations to create, manage and resolve decentralized identifiers.
+- Pollux: Provides a suite of operations for handling [verifiable credentials](https://github.com/hyperledger/identus-docs/blob/master/documentation/docs/concepts/glossary.md#verifiable-credentials).
+- Mercury: Provides a suite of operations for handling DIDComm V2 messages.
+- Pluto: Provides an interface for storage operations in a portable, storage-agnostic manner.
+- Agent: A component using all other building blocks, provides basic edge agent capabilities, including implementing DIDComm V2 protocols.
```

Comment:
Do we need to even talk about the building blocks in the Wallet section?  We already mention them in the Overview.  Do these really apply to "wallets"?  Or maybe we should have a Section for "Edge Agents"?  I'm not sure it's clear to the reader what the difference between a "Wallet" and an "Edge Agent" is